team.html

```diff
@@ -74,30 +86,38 @@
 	  		<div class="col-md-2">
 	  			<img src="https://alix-tz.github.io/assets/images/profile.png" class="img-fluid rounded" alt="Photo d'Alix Chagué" />
 	  		</div>
 	  		<div class="col-offset-1 col text-justify">
 	  			<h2><!--<span aria-ssml="{&quot;phoneme&quot;:{&quot;ph&quot;:&quot;a.liks ʃaɡe&quot;,&quot;alphabet&quot;:&quot;ipa&quot;}}">-->Alix Chagué</span> <!--(\a.liks ʃaɡe\)--></h2>
+          <p><a class="mt-2 btn btn-sm btn-outline-success" href="https://orcid.org/0000-0002-0136-4434" target="_blank"><i class="fab fa-orcid"></i> 0000-0002-0136-4434</a></p>
 	  			<p><i18n vanilla-i18n="team.alix">Je suis doctorante en Humanités numérique au sein des équipes ALMAnaCH du centre Inria de Paris et celle du CRIHN de l'Université de Montréal, en cotutelle EPHE-Université de Montréal. Mes recherches portes sur l'appropriation des outils de transcription automatique (HTR) par la communauté des humanités numériques. Initialement issue d'une formation classique en sciences humaines et en histoire de l'art, je suis co-responsable du Master 2 Documentation et Humanités Numériques de l'Ecole du Louvre, où je dispense des enseignements méthodologiques et techniques. En tant qu'ingénieure R&D pour ALMAnaCH, j'ai notamment contribué aux projets TIMEUS (ANR), LECTAUREP et DAHN. </i18n></p>
 	  		</div>
 	  	</section>
 	  	<hr />
-	  	<section class="row" id="thibault">
-	  		<div class="col-offset-1 col  text-justify">
+	  	<section class="row py-4" id="thibault">
+	  		<div class="col-offset-1 col text-justify">
 	  			<h2>Thibault Clérice</h2>
+          <p><a class="mt-2 btn btn-sm btn-outline-success" href="https://orcid.org/0000-0003-1852-9204" target="_blank"><i class="fab fa-orcid"></i> 0000-0003-1852-9204</a></p>
 	  			<p><i18n vanilla-i18n="team.thibault">Je suis responsable du master « Technologies numériques appliquées à l’histoire » de l’École nationale des chartes (Paris, France). J'ai été ingénieur au Centre for eResearch (Kings College de Londres, Royaume-Uni) et à la chaire Humboldt pour les humanités numériques (Leipzig, Allemagne), où j'ai travaillé sur les fondations de Perseus 5 et du projet CapiTainS.org. Je suis membre fondateur du Technical Committee du standard Distributed Text Services ( w3id.org/dts ). Mes enseignements portent en majeure partie sur l’ingénierie des données patrimoine, qu’il s’agisse de la gestion numérique des données, des bonnes pratiques de développement, de standards de communication ou de langages de programmation. Mes recherches portent principalement sur le traitement automatique des langues anciennes à travers l’apprentissage profond et sur la mise à disposition de corpora.</i18n></p>
 	  		</div>
 	  		<div class="col-md-2">
 	  			<img src="https://pbs.twimg.com/profile_images/922043022916902913/Xdvaypzu_400x400.jpg" class="img-fluid rounded" alt="Photo de Thibault Clérice" />
 	  		</div>
 	  	</section>
+
+      <h1 class="mt-4 mb-3">Publications</h1>
+      <ul class="list-group">
```

Comment:
Le texte n'est pas visible chez moi à cause du footer ? 
![image](https://user-images.githubusercontent.com/1929830/171929655-0ef735c7-fdb5-42e6-b806-577e5e7a27a5.png)

Reply:
Then I'll remove the sticky footer on that page

Reply:
I mean I just want it to stay at the bottom of the page in stead of being in the middle when the page is too short...